risc0/zkvm/src/host/protos/api.proto

```diff
@@ -12,7 +12,7 @@ message ServerRequest {
     ProveSegmentRequest prove_segment = 3;
     LiftRequest lift = 4;
     JoinRequest join = 5;
-    IdentityP254Request identiy_p254 = 6;
+    IdentityP254Request identity_p254 = 6;
```

Comment:
My experience with "proto" is that changing the name does not break the compatibility, and the identity-p254-request is not in the public release. Please check if this is the case.

Reply:
Yes and no. It breaks if the encoding is JSON, but at this time we don't use any JSON encodings, so this should be fine.